### DISCUSSION TITLE
Support Route Groups for pages router

### DISCUSSION THREAD

**Discussion by MauriceArikoglu** [vercel/next.js#56160] [Ideas]
### Goals

1. Feature parity between app and pages router
2. Better project organization 
3. More clarity in the use of framework intricacies

### Non-Goals

1. Change features
2. 
3.

### Background

Currently there is no way to organize a large project consistently when using the pages router.

It makes sense to introduce path grouping features to the pages router if there are paths that logically belong together but shouldn’t be grouped under actual paths.

I don’t see a reason this feature shouldn’t be implemented.

Currently, a folder in pages router named `[something]` will act the same as a folder named `[something]` in the app router.

In contrast, a folder named `(something)` in app router ends up ignored, while a folder named `(something)` in pages router will lead to the path `…/(something)` - which is a structure I’ve never seen used in the world wild web.

Therefore I assume it is a case rarely used and fine to introduce as breaking change in a minor version update.

### Proposal

Align the app and pages router compilers to treat folders wrapped with parentheses equally as path group.

**Comment by leerob** [comment]
Thanks for the suggestion! We are not planning to add support for Route Groups to the Pages Router, but would recommend exploring [incrementally adopting the App Router](https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration) for this. While there will be some new features added the Pages Router in the coming months, this is not one of them. Thank you.
